Amazon.QLDB.Driver/driver/AsyncQldbDriver.cs

```diff
@@ -45,14 +46,17 @@ public class AsyncQldbDriver : IAsyncQldbDriver
         /// <param name="sessionClient">AWS SDK session client for QLDB.</param>
         /// <param name="maxConcurrentTransactions">The maximum number of concurrent transactions.</param>
         /// <param name="logger">The logger to use.</param>
+        /// <param name="serializer">The serializer to serialize and deserialize Ion data.</param>
         internal AsyncQldbDriver(
             string ledgerName,
             IAmazonQLDBSession sessionClient,
             int maxConcurrentTransactions,
-            ILogger logger)
+            ILogger logger,
+            ISerializer serializer = null)
```

Comment:
This seems very odd.

We have a builder to set defaults... so why not set the default in the builder?

Reply:
The [builder](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/object-serialization-idda-jun-4-2021/Amazon.QLDB.Driver/driver/BaseQldbDriverBuilder.cs#L29) is setting serializer to `default`.

Removing this because it is redundant